include/zephyr/dt-bindings/pinctrl/stm32-pinctrl.h

```diff
@@ -108,9 +108,9 @@
 #define STM32_PUPDR_SHIFT		9
 
 /* GPIO plain output value */
-#define STM32_ODR_0			(0x0 << STM32_ODR_SHIFT)
-#define STM32_ODR_1			(0x1 << STM32_ODR_SHIFT)
-#define STM32_ODR_MASK			0x1
-#define STM32_ODR_SHIFT			11
+#define STM32_ODR_0			(0x1 << STM32_ODR_SHIFT)
+#define STM32_ODR_1			(0x2 << STM32_ODR_SHIFT)
+#define STM32_ODR_MASK			0x3
+#define STM32_ODR_SHIFT			12
```

Comment:
We should keep in ODR shift the number of places.

Reply:
good catch, txs